webcomponents/search-webcomponent/src/app/mp-search.component.ts

```diff
@@ -26,13 +33,20 @@ export class MpSearchComponent implements OnInit {
 
   title = 'search-webcomponent';
   public appLoaded: boolean = true;
+  public appLanguage: string;
   private currentCategoryId = 1;
+  @Input() language: string;
 
   constructor(
-    private readonly translate: TranslateService,
-    private store: Store<MarketplaceState>
-  ) {
-    translate.setDefaultLang('de');
+    private store: Store<MarketplaceState>,
+    private localization: LocalizationService
+  ) { }
+
+  ngOnChanges(changes: SimpleChanges) {
```

Comment:
Might as well implement the OnChanges interface to get rid of this warning, similar to previous issue.

Reply:
done